odoo/addons/base/models/res_users.py

```diff
@@ -1551,8 +1553,7 @@ def _get_group_definitions(self):
         return SetDefinitions(data)
 
 
-class UsersImplied(models.Model):
-    _name = 'res.users'
+class ResUsers(models.Model):
```

Comment:
@Gorash During review, I found this change (and the one in `ir_model.py` which was probably introduced when running the migration script. Shouldn't there be an exception to avoid changing these base classes?

-- after a first review, I will rollback these 2 changes